velocity_estimation_node:
  ros__parameters:
    frequency: 40         # [Hz]

    initial_state_vector: [0.0, 0.0, 1.0, 0.5, 0.5, -0.5]

    initial_state_covariance: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    
    process_noise_covariance: [0.05, 0.0, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.05, 0.0, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.001, 0.0, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.1, 0.0, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.1, 0.0,
                               0.0, 0.0, 0.0, 0.0, 0.0, 0.01]

    #measurement_noise_covariance: 

    # Sensor Positions
    vectornav-vn-200:
      x: 0.7
      y: 0.1
    vectornav-vn-300:
      x: -0.5
      y: 0.1